krabs/krabs/schema_locator.hpp

```diff
@@ -32,28 +32,84 @@ namespace krabs {
     struct schema_key
     {
         guid      provider;
+
+        // Using a string_view for name so that keys can be constructed from
```

Comment:
super-nit: use a consistent format for block comments (/** */ in this file)

Reply:
Updated.